shortfin/python/shortfin_apps/flux/simple_client.py

```diff
@@ -24,39 +29,17 @@
     "neg_prompt": ["Watermark, blurry, oversaturated, low resolution, pollution"],
     "height": [1024],
     "width": [1024],
-    "steps": [2],
+    "steps": [20],
     "guidance_scale": [3.5],
     "seed": [0],
     "output_type": ["base64"],
     "rid": ["string"],
 }
 
 
-def bytes_to_img(in_bytes, outputdir, idx=0, width=1024, height=1024):
-    timestamp = dt.now().strftime("%Y-%m-%d_%H-%M-%S")
-    image = Image.frombytes(
-        mode="RGB", size=(width, height), data=base64.b64decode(in_bytes)
-    )
-    if not os.path.isdir(outputdir):
-        os.mkdir(outputdir)
-    im_path = os.path.join(outputdir, f"shortfin_flux_output_{timestamp}_{idx}.png")
-    image.save(im_path)
-    print(f"Saved to {im_path}")
-
-
-def get_batched(request, arg, idx):
-    if isinstance(request[arg], list):
-        # some args are broadcasted to each prompt, hence overriding idx for single-item entries
-        if len(request[arg]) == 1:
-            indexed = request[arg][0]
-        else:
-            indexed = request[arg][idx]
-    else:
-        indexed = request[arg]
-    return indexed
-
-
-async def send_request(session, rep, args, data):
+async def send_request(
+    session: aiohttp.ClientSession, rep: int, args: argparse.Namespace, data: dict
```

Comment:
I think there's an opportunity for a separate 30sec PR right here! I also have some more thoughts on this, but don't wanna clog up this review haha.

Could you bubble this signature change to its own commit at the beginning of this branch?

Reply:
This seems unnecessary given the commits will be squashed before merge.

Reply:
On that basis, totally unnecessary, for sure! I'd say it's more about PR granularity, diff size, merge conflict avoidance, etc. We can skip it for now!

The thing I was gonna mention is making the arguments multiline for easier diffing:
```suggestion
    session: aiohttp.ClientSession,
    rep: int,
    args: argparse.Namespace,
    data: dict,
```

Reply:
Yeah that seems reasonable.

Reply:
The pre-commit actually undoes this change... let's keep aligned with the automatic linter in this case.

Reply:
Just tried it, and it seems to only undo the change if you leave off the dangling comma! Does it pass pre-commit for you if you put the comma after `dict`?